### DISCUSSION TITLE
1.6.0.Final

### DISCUSSION THREAD

**Discussion by alantanlc** [mapstruct/mapstruct#3596] [General]
Is there a release date for 1.6.0.Final?

**Comment by filiphr** [comment]
No we currently do not have a fix release date for 1.6.0.Final. We released 1.6.0.Beta2 2 weeks ago. Next would be the RC and then the final. I'm hopeful that we can release sooner as we've automatized our release now, which makes it easier.

I'm traveling in the next month, so that would be a bit more difficult, but hopefully some more activity around July / August

**Reply by tragait** [reply]
Hi @filiphr 

how are you doing? Any update on RC/Final release for 1.6.0.

Thanks,
Rahul

**Reply by filiphr** [reply]
Hey @tragait,

We just released 1.6.0.RC1 (https://mapstruct.org/news/2024-07-20-mapstruct-1_6_0_RC1-is-out/). The RC looks good and I hope that in the next 2 weeks we will release the final release as well.
